lib/socket.ex

```diff
@@ -68,7 +68,7 @@ defmodule Channels.Client.Socket do
   you can simply return `{:stop, reason, state}` from any of the callback.
   """
   use GenServer
-  alias Channels.Client.WebsocketTransport
+  alias Phoenix.Channels.GenSocketClient.WebSocketClient
```

Comment:
Isn't `WebSocket` generally written `websocket`, and hence when capitalised `Websocket`?
Obviously it is a socket over a web transport channel so the camel-casing is fine too, but just asking.